common/src/main/java/org/apache/rocketmq/schema/registry/common/QualifiedName.java

```diff
@@ -80,18 +80,17 @@ public String subjectFullName() {
 
     @Override
     public String toString() {
-        final StringBuilder sb = new StringBuilder("{");
-        sb.append("\"cluster\":\"")
```

Comment:
why change this？

Reply:
The original NotExistException uses `qualifiedName.fullname()` to return error message, so the error parameter is not clear from the `fullname`, so I change it to `toString`. 
When using `toString()` to return error message, the HTTP will fail to parse the error message maybe because of containing quotation marks, so I change it to non JSON format.